### DISCUSSION TITLE
DoT and DoH at the same time in Blocky

### DISCUSSION THREAD

**Discussion by m0zgen** [0xERR0R/blocky#302] [Q&A] [answered]
Hello, blocky works only with one port 443 or 853 with enabled https parameter. As possible - enable both at the same time?

**Comment by 0xERR0R** [comment, marked as answer]
Hi,

currently, blocky can serve only TCP/UPD (default port 53) or DoH (default port 443). DoT is not implemented yet. But it was on my personal todo list and I created #303 to track it. Since Android supports only DoT natively, it makes sense to implement it.

**Reply by m0zgen** [reply]
Thanks, sound good.
